expression/builtin_compare_vec_test.go

```diff
@@ -44,6 +44,11 @@ var vecBuiltinCompareCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt, types.ETInt, types.ETInt}},
 		{retEvalType: types.ETReal, childrenTypes: []types.EvalType{types.ETReal, types.ETReal, types.ETReal}},
 	},
+	ast.Interval: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETReal, types.ETReal}},
```

Comment:
In mysql5.7, the function ```INTERVAL(N,N1,N2,N3,...)``` needs to meet the following conditions that ```N < N1< N2 < N3```. See [here](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#function_interval) for more detail.